engine/execution/computation/manager.go

```diff
@@ -130,7 +167,9 @@ func (e *Manager) ComputeBlock(
 func (e *Manager) GetAccount(address flow.Address, blockHeader *flow.Header, view *delta.View) (*flow.Account, error) {
 	blockCtx := fvm.NewContextFromParent(e.vmCtx, fvm.WithBlockHeader(blockHeader))
 
-	account, err := e.vm.GetAccount(blockCtx, address, view)
+	programs := e.getProgramsOrEmpty(blockHeader.ID())
+
+	account, err := e.vm.GetAccount(blockCtx, address, view, programs)
```

Comment:
same comment as scripts, is it safe to use the same program for transactions and accounts?

Reply:
As we control all the inputs here, I would say yes